pkg/driver/bacnet/config/traits.go

```diff
@@ -30,6 +30,9 @@ type Trait struct {
 	PollPeriod *Duration `json:"pollPeriod,omitempty"`
 	// how long to wait when running a poll for it to respond. Defaults to PollPeriod.
 	PollTimeout *Duration `json:"pollTimeout,omitempty"`
+	// maximum period for tries for the test during a poll event
+	// defaults to 1m to not waste server resources
+	MaxTriesDelay *Duration `json:"maxTriesDelay,omitempty"`
```

Comment:
I don't think the name or comment clearly describe what this property does.

Maybe this is clearer?
```go
// after a write, how long to wait before giving up on reads becoming consistent with the written value.
RWConsistencyTimeout *Duration `json:"rwConsistencyTimeout,omitempty"`
```